### DISCUSSION TITLE
Winget wants to downgrade 1.2.3-1 to 1.2.3

### DISCUSSION THREAD

**Discussion by tawissus** [rustdesk/rustdesk#7490] [General]
### Bug Description

winget uprade:

Name             ID                             Version      Available      Quelle
RustDesk        RustDesk.RustDesk   1.2.3-1       1.2.3             winget

Winget now downloads and installs the old version from this page:
https://github.com/rustdesk/rustdesk/releases/download/1.2.3/rustdesk-1.2.3-x86_64.exe

### How to Reproduce

Run "winget upgrade" or "winget upgrade --all" with Version 1.2.3-1.
Winget offer/install a "new" Version 1.2.3.

### Expected Behavior

Don't offer/install older versions. 

### Operating system(s) on local side and remote side

Windows 10/11

### RustDesk Version(s) on local side and remote side

1.2.3-1

### Screenshots

<img width="622" alt="pwsh_s66rkizomb" src="https://github.com/rustdesk/rustdesk/assets/1453423/497bc55c-06e6-4b3f-8f84-834ddb2d73ac">

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/22da14e3f70e14ab7c87db483d4470b7f8ffdeef
